service/src/main/java/uk/gov/hmcts/reform/fpl/enums/ComplyOnBehalfEvent.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+public enum ComplyOnBehalfEvent {
```

Comment:
Would change this to be `Event` or `EventID`. This then allows for it to be added to in the future if we need to implement something similar

Reply:
good idea, let's deal with this when a requirement comes up though? What do you think?

Reply:
Sounds good to me